core/static/js/opencanada.js

```diff
@@ -43,6 +43,9 @@ function initForWindow(){
 
 $(window).resize(function(){
     initForWindow();
+    windowHeight = $(window).height();
```

Comment:
If we access the properties directly, we shouldn't need to update these variables.